frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown"></TextBlock>
+        <div class="flex justify-between mt-4">
+            <span>Current global limit</span>
+            <span v-if="liquidationLimits"
```

Comment:
As discussed, this should never be the case, instead, please check each key in a computed prop